app/services/metrics/review_turnaround/per_user_project.rb

```diff
@@ -36,8 +64,18 @@ def calculate_turnaround(review)
         review.opened_at.to_i - review.review_request.created_at.to_i
       end
 
-      def create_metric(entity, turnaround)
-        Metric.create!(ownable: entity, value: turnaround, name: :review_turnaround)
+      def create_or_update_metric(entity, turnaround)
+        metric = Metric.find_or_initialize_by(ownable: entity,
+                                              created_at: Time.zone.today.all_day,
+                                              name: :review_turnaround)
+        return metric.update!(value: (turnaround + metric.value)) if metric.persisted?
+
+        metric.update!(value: turnaround)
```

Comment:
you are doing this because `metric.value` can be `nil` ? If yes:
maybe we can make that column not nullable
or
you can do `metric.update!(value: (turnaround + metric.value.to_i))` because `nil.to_i == 0`

Reply:
or `to_f` instead of `to_i` depending on the data type

Reply:
No, I made it in that way just to skip this:
```
metric.value = turnaround
metric.save!
```